packages/styles/scripts/autolayout.ts

```
@@ -68,8 +73,8 @@ const createBodyAutolayoutSCSS = (spacingMatrix: DefaultSpacingMatrix): string =
     let selector = '';
 
     if (insertType === INSERT_SPACE_BETWEEN) {
-      if (!component2 || typeof component2 === 'number') {
-        selector = `.${component.toLowerCase()}`;
+      if (!component2 || typeof component2 === null) {
```

Biome

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)